results/cmd/watcher/main.go

```diff
@@ -24,19 +24,23 @@ import (
 	"github.com/tektoncd/experimental/results/pkg/convert"
 	pb "github.com/tektoncd/experimental/results/proto/proto"
 	"github.com/tektoncd/pipeline/pkg/apis/pipeline"
+
 	taskruninformer "github.com/tektoncd/pipeline/pkg/client/injection/informers/pipeline/v1beta1/taskrun"
 	listers "github.com/tektoncd/pipeline/pkg/client/listers/pipeline/v1beta1"
+
 	"go.uber.org/zap"
 	"google.golang.org/grpc"
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
```

Comment:
What is this used for? (as well as knative.dev/pkg/system/testing)

Reply:
Without this import, when I ran the watcher.go, in gave me an error

Reply:
Done